web/src/components/LandingPageSlider.tsx

```diff
@@ -0,0 +1,176 @@
+import { LandingPageActionTile } from "@/components/LandingPageActionTile";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { QUERIES, ROUTES, routeWithQuery } from "@/lib/domain-logic/routes";
+import analytics from "@/lib/utils/analytics";
+import { useRouter } from "next/router";
+import { ReactElement, useRef, useState } from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick-theme.css";
+import "slick-carousel/slick/slick.css";
+import { useMountEffect } from "../lib/utils/helpers";
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+function SampleNextArrow(props: any) {
```

Comment:
we should not be using "any" unless there really is no other choice. In this case, there's a lot of options for providing types, but the easiest is to just use the types provided by the library.

I went ahead and inspected the definition of the "Slider" element, which showed me its type definition, and the type definitions in the Settings and in the library. From there, it was easy to see that the library exports a type called "CustomArrowProps" which is exactly what you should be using here